core/scrollbar.js

```diff
@@ -181,6 +179,7 @@ Blockly.ScrollbarPair.prototype.getRatio_ = function(knobValue, barLength) {
  * Class for a pure SVG scrollbar.
  * This technique offers a scrollbar that is guaranteed to work, but may not
  * look or behave like the system's scrollbars.
+ * NB: "Knob" and "handle" are used interchangeably in this file.
```

Comment:
It seems like a pretty obscure thing to reference.  I vote to rename.

Send (awkwardly) from an Android phone.
On 27 May 2016 9:34 a.m., "rachel-fenichel" <notifications@github.com>
wrote:

> In core/scrollbar.js
> <https://github.com/google/blockly/pull/382#discussion_r64932240>:
>
> > @@ -181,6 +179,7 @@ Blockly.ScrollbarPair.prototype.getRatio_ = function(knobValue, barLength) {
> >   * Class for a pure SVG scrollbar.
> >   * This technique offers a scrollbar that is guaranteed to work, but may not
> >   * look or behave like the system's scrollbars.
> > + * NB: "Knob" and "handle" are used interchangeably in this file.
>
> TBH, mostly fear that someone, somewhere is using private properties of
> the scrollbar.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/blockly/pull/382/files/5752913b4d0b14d427d78a760046fb5903b8808a#r64932240>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFgpv84ecVe-pQqgSUWbSx22EX5y3R7zks5qFx0RgaJpZM4InD5Z>
> .
>